src/components/display/Table.tsx

```diff
@@ -203,7 +203,7 @@ const DefaultHeaderFactory: React.VFC<THeaderProps> = ({
 			<th align="center">
 				{showCheckbox && multiSelect && (showCkb || selectionMode || allSelected) && (
 					<Checkbox
-						iconSize="medium"
+						size={'medium'}
```

Comment:
The previous icon was smaller, I think medium was referring to older DS measures